app/views/partials/lib/commands/users/create/check.liquid

```diff
@@ -14,6 +14,10 @@
     include 'lib/register_error', contract: c, field_name: 'password', key: 'app.errors.blank'
   endif
 
+  if object.agreed == blank
```

Comment:
@damcik Please use defined validation partial
`  function c = 'lib/validations/presence', c: c, object: object, field_name: 'agreed'`

You would have to add `key` argument to this function and pass your own translation key.